src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java

```diff
@@ -56,12 +54,13 @@ public void blockedUserShouldSeeMessageOnArticleEdit() {
   @Test(groups = {"usersAndRights003"}, dependsOnMethods = {"staffCanBlockUser"})
   @Execute(asUser = User.BLOCKED_USER)
   public void blockedUserShouldBeAbleToChangeEmail() {
+    String username = Configuration.getCredentials().emailQaart2;
+    String password = Configuration.getCredentials().emailPasswordQaart2;
```

Comment:
For the future: Perhaps there can be a `User Configuration.getCredentials()` function?